bridges/modules/xcm-bridge-hub/src/lib.rs

```diff
@@ -56,63 +202,1513 @@ pub mod pallet {
 		/// `BridgedNetworkId` consensus.
 		type BridgeMessagesPalletInstance: 'static;
 
-		/// Price of single message export to the bridged consensus (`Self::BridgedNetworkId`).
+		/// Price of single message export to the bridged consensus (`Self::BridgedNetwork`).
 		type MessageExportPrice: Get<Assets>;
 		/// Checks the XCM version for the destination.
 		type DestinationVersion: GetVersion;
 
-		/// Get point-to-point links with bridged consensus (`Self::BridgedNetworkId`).
-		/// (this will be replaced with dynamic on-chain bridges - `Bridges V2`)
-		type Lanes: Get<sp_std::vec::Vec<(SenderAndLane, (NetworkId, InteriorLocation))>>;
-		/// Support for point-to-point links
-		/// (this will be replaced with dynamic on-chain bridges - `Bridges V2`)
-		type LanesSupport: XcmBlobHauler;
+		/// The origin that is allowed to call privileged operations on the pallet, e.g. open/close
+		/// bridge for location that coresponds to `Self::BridgeOriginAccountIdConverter` and
+		/// `Self::BridgedNetwork`.
+		type AdminOrigin: EnsureOrigin<<Self as SystemConfig>::RuntimeOrigin>;
+		/// A set of XCM locations within local consensus system that are allowed to open
+		/// bridges with remote destinations.
+		type OpenBridgeOrigin: EnsureOrigin<
+			<Self as SystemConfig>::RuntimeOrigin,
+			Success = Location,
+		>;
+		/// A converter between a location and a sovereign account.
+		type BridgeOriginAccountIdConverter: ConvertLocation<AccountIdOf<ThisChainOf<Self, I>>>;
+
+		/// Amount of this chain native tokens that is reserved on the sibling parachain account
+		/// when bridge open request is registered.
+		#[pallet::constant]
+		type BridgeDeposit: Get<BalanceOf<ThisChainOf<Self, I>>>;
+		/// Currency used to pay for bridge registration.
+		type Currency: MutateHold<
```

Comment:
I'd call it `Fungible` to not confuse it with the old Currency traits, which we still have around 😅

Reply:
well, I used `Currency`, because it is used everywhere. I only see one `type Fungibles:`, but no `type Fungible:`. So, I think it would be better to rename this in all other places as well, but as a separate PR.